### DISCUSSION TITLE
IPED does not render a specific .eml file

### DISCUSSION THREAD

**Discussion by PCF-03** [sepinf-inc/IPED#953] [Show and tell]
Esteemed,

I processed several Microsoft mailboxes and IPED is not reading the .eml emails. Does not visualize. What can it be? Do you know the solution??
I am in need of a solution.

Thanks!

Luis Marcos

**Comment by lfcnassif** [comment]
What is your IPED version? Could you share the processing and the analysis log? Could you share a problematic EML file to reproduce? Privatelly if it has sensitive data...

**Comment by PCF-03** [comment]
Hi Nasif,

I'm using IPED version 3.18.12 (IPED-3.18.12_and_plugins.zip). I downloaded from this link https://github.com/sepinf-inc/IPED/releases/.
It's confidential data. But I can send you the logs and some .emls files privately. How do I send them??

Thanks for the quick response! 🙏

**Reply by paulobreim** [reply]
My suggestion is run again with the last version.

**Reply by PCF-03** [reply]
You mean processing everything again??
I would not like.

**Reply by lfcnassif** [reply]
@PCF-03 you can send them to the email on my profile. I can take a look on late Monday when I return from vacation. If you could share publicly, maybe other developer or user could help you before that...

**Reply by PCF-03** [reply]
Nassif,

I will send the IPED logs to your gmail box.
If you can analyze and give feedback I would appreciate it.

**Reply by lfcnassif** [reply]
Looking at the logs sent by @PCF-03, they are from 3.18.9, not 3.18.12. There is an issue with a a single EML with invalid NTFS chars in attachment names, not with all EMLs. I opened #954. I'm waiting @PCF-03 to test the fix, since a triggering file was not shared for testing.

**Reply by lfcnassif** [reply]
Actually, the issue @PCF-03 was complaining about was probably #785, already fixed in 3.18.10, since the issue arises in a different computer than the processing machine (detail not explained here...).
